vignettes/hubEnsembles.Rmd

```diff
@@ -18,167 +18,325 @@ knitr::opts_chunk$set(
+  model_id = c("MOBS-GLEAM_FLUH", "PSI-DICE", "simple_hub-baseline"),
   weight = c(0.4, 0.4, 0.2)
 )
-
-weighted_mean_ens <- hubEnsembles::simple_ensemble(model_outputs,
-  weights = model_weights,
-  model_id = "hub-ensemble-weighted-mean"
-)
-head(weighted_mean_ens)
+weighted_mean_ens <- hubExamples::forecast_outputs |>
+  dplyr::filter(output_type != "sample") |>
+  hubEnsembles::simple_ensemble(
+    weights = model_weights,
+    model_id = "simple-ensemble-weighted-mean"
+  )
+head(weighted_mean_ens, 10)
 ```
 
 # Creating ensembles with `linear_pool`
 
-An alternative approach to generate an ensemble is a linear pool, or a distributional mixture; this function can be applied to predictions with an `output_type` of `mean`, `quantile`, `cdf`, or `pmf`. Our example hub includes `median` output type, so we exclude it from the calculation.
+The `linear_pool()` function implements the linear opinion pool (LOP, also known as a distributional mixture) method when ensembling predictions. This function can be used to combine predictions with output types mean, quantile, CDF, and PMF. Unlike `simple_ensemble()`, this function handles its computation differently based on the output type. For the CDF, PMF, and mean output types, the linear pool method is equivalent to calling `simple_ensemble()` with a mean aggregation function, since `simple_ensemble()` produces a linear pool prediction (an average of individual model cumulative or bin probabilities).
 
-For `mean`, `cdf` and `pmf` output types, the linear pool is equivalent to using a mean `simple_ensemble`. For `quantile` model outputs, the `linear_pool` function needs to approximate a full probability distribution using the value-quantile pairs from each component model. As a default, this is done with functions in the `distfromq` package, which defaults to fitting a monotonic cubic spline.
+For the quantile output type, the `linear_pool()` function first must approximate a full probability distribution using the value-quantile level pairs from each component model. As a default, this is done with functions in the `distfromq` package, which defaults to fitting a monotonic cubic spline for the interior and a Gaussian normal distribution for the tails. Quasi-random samples are drawn from each distributional estimate, which are then collected and used to extract the desired quantiles from the final ensemble distribution.
+
+Using the default options for `linear_pool()`, we can generate an equally-weighted linear pool for each of the output types in our example hub (except for the median and sample output types, which must be excluded). The resulting distribution for the linear pool of quantiles is estimated using a default of `n_samples = 1e4` quasi-random samples drawn from the distribution of each component model.
 
 ```{r}
-linear_pool_ens <- model_outputs %>%
-  filter(output_type != "median") %>%
-  hubEnsembles::linear_pool()
-head(linear_pool_ens)
+options(width = 300)
```

Comment:
If it's possible and easy, can we put this option in the `{r}` call instead? I did some brief searching and couldn't figure out how, but perhaps you know @lshandross since you were digging around on this? Don't think it's worth spending a lot of time on.